user-service/src/main/java/uk/gov/gchq/palisade/service/user/stream/PropertiesConfigurer.java

```diff
@@ -89,7 +90,7 @@ public void afterPropertiesSet() {
         MutablePropertySources envPropSources = ((ConfigurableEnvironment) this.environment).getPropertySources();
         envPropSources.forEach((PropertySource<?> propertySource) -> {
             if (propertySource.containsProperty("application.properties.locations")) {
-                locations = ((String) propertySource.getProperty("application.properties.locations"))
+                locations = ((String) Objects.requireNonNull(propertySource.getProperty("application.properties.locations")))
```

Comment:
We should have a preference for `Optional.ofNullable(...).orElseThrow(() -> new PropertyLoadingException("..."))` if this must be non-null

Reply:
Replaced